ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/PartyManagementServiceAnnotationsValidationTests.scala

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites.v1_8
+
+import java.nio.charset.StandardCharsets
+
+import com.daml.error.definitions.LedgerApiErrors
+import com.daml.ledger.api.testtool.infrastructure.Allocation.{
+  NoParties,
+  Participant,
+  Participants,
+  allocate,
+}
+import com.daml.ledger.api.testtool.infrastructure.Assertions.{assertEquals, _}
+import com.daml.ledger.api.v1.admin.object_meta.ObjectMeta
+import com.daml.ledger.api.v1.admin.party_management_service.{
+  AllocatePartyRequest,
+  PartyDetails,
+  UpdatePartyDetailsRequest,
+}
+import com.google.protobuf.field_mask.FieldMask
+
+trait PartyManagementServiceAnnotationsValidationTests { self: PartyManagementServiceIT =>
+
+  test(
+    "PMTestAnnotationsSizeLimits",
+    "Test annotations' size limit",
+    enabled = features => features.userAndPartyManagementExtensionsForHub,
+    partyAllocation = allocate(NoParties),
+  )(implicit ec => { case Participants(Participant(ledger)) =>
+    val largeString = "a" * 256 * 1024
```

Comment:
Can we tie this entire test to a constant defining the limit in the implementation class?

Reply:
Ok - defining a constant but in this trait directly